ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -6,13 +6,13 @@ let prototyped_of_field = function
   | "Repository", "gpgkey_path" ->
       Some "22.12.0"
   | "host", "last_software_update" ->
-      Some "22.19.0-next"
```

Comment:
Is this related or accidental?

Reply:
This is a change that is forced by the versioning mechanism we have, everytime the code is compile this source file is changes, which might make it to unrelated commit if we're not careful, hence why I added it a separate commit. This is the same change as in Rob's PR

Reply:
Yeah, it was actually in 22.20.0 but got missed. I've got the right version in my PR, which was merged, so this change should now be removed.